docs/golden-path/create-app/local-development.md

```diff
@@ -0,0 +1,57 @@
+---
+id: local-development
+title: 002 - Local development
+---
+
+Your Backstage app is fully installed and ready to be run! Now that the installation is complete, you can go to the application directory and start the app using the `yarn start` command. The `yarn start` command will run both the frontend and backend as separate processes (named `[0]` and `[1]`) in the same window.
+
+```bash
+cd my-backstage-app # your app name
+yarn start
+```
+
+![Screenshot of the command output, with the message webpack compiled successfully](../../assets/getting-started/startup.png)
+
+Here again, there's a small wait for the frontend to start up. Once the frontend is built, your browser window should automatically open.
+
+:::tip Browser window didn't open
+
+When you see the message `[0] webpack compiled successfully`, you can navigate directly to `http://localhost:3000` to see your Backstage app.
+
+:::
+
+You can start exploring the demo immediately.
+
+![Screenshot of the Backstage portal](../../assets/getting-started/portal.png)
```

Comment:
Do we need this? Otherwise maybe we reword it to something like

```suggestion
Once its spun up, you should see something similar to the below.

![Screenshot of the Backstage portal](../../assets/getting-started/portal.png)
```